deployments/helm/cvmfs-csi/templates/prefetcher-configmaps.yaml

```
@@ -11,7 +11,7 @@ metadata:
 data:
   # Entrypoint for cvmfs-csi prefetcher job mounted at 
   # /etc/cvmfs-csi/prefetcher/entrypoint.sh.
-  entrypoint.sh: {{ toYaml .script | nindent 4 }}
+  entrypoint.sh: {{ tpl .script $ | nindent 4 }}
```

```suggestion
  entrypoint.sh: |
    {{- tpl .script $ | nindent 4 }}
```

This is to ensure that newlines are processed properly by helm.

Before the change templating with the mock jobs provided in the values gives an invalid entrypoint script if your script is over many lines. (I think this works in the case of your example as all your commands are seperated by two newlines.)

```yaml
apiVersion: v1
data:
  cron.d: |
    SHELL=/bin/bash
    PATH=/sbin:/bin:/usr/sbin:/usr/bin
    MAILTO=""
    * * * * * root bash /etc/cvmfs-csi/prefetcher/entrypoint.sh
  entrypoint.sh: echo -e "prefetching sft.cern.ch" ls /cvmfs/sft.cern.ch
kind: ConfigMap
metadata:
  labels:
    ...
  name: cvmfs-cvmfs-csi-prefetcher-example-job
  namespace: cvmfs
```
Whereas with the changes you get
```yaml
apiVersion: v1
data:
  cron.d: |
    SHELL=/bin/bash
    PATH=/sbin:/bin:/usr/sbin:/usr/bin
    MAILTO=""
    * * * * * root bash /etc/cvmfs-csi/prefetcher/entrypoint.sh
  entrypoint.sh: |
    #!/bin/bash
    echo -e "prefetching sft.cern.ch"
    ls /cvmfs/sft.cern.ch
kind: ConfigMap
...
```